src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,172 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+function preperedPeople(
+  query: string,
+  filterField: string,
+  centuries: string[],
+  sortField: string,
+  sortOrder: string,
+  readyPeople: Person[],
+) {
+  let visiblePeople = [...readyPeople];
+  const preperedQuery = query.trim().toLowerCase();
+
+  if (sortField) {
+    visiblePeople.sort((a: Person, b: Person) => {
+      switch (sortField) {
+        case 'name':
+          if (sortOrder === 'asc') {
+            return a.name.localeCompare(b.name);
+          } else if (sortOrder === 'desc') {
+            return b.name.localeCompare(a.name);
+          } else {
+            return 0;
+          }
+
+        case 'sex':
+          if (sortOrder === 'asc') {
+            return b.sex.localeCompare(a.sex);
+          } else if (sortOrder === 'desc') {
+            return a.sex.localeCompare(b.sex);
+          } else {
+            return 0;
+          }
```

Comment:
logic for string values are the same in different cases
You can join cases:

```
case 'name':
case 'sex':
  // logic here b[sortField].localCompare
```
Same for numeric fields